pom.xml

```diff
@@ -59,6 +59,7 @@
     <!-- We skip sources jar generation as we do it with the assembly plugin to have greater control over the content -->
     <source.skip>true</source.skip>
     <kotlin.version>1.3.0</kotlin.version>
+    <testcontainers.version>1.12.1</testcontainers.version>
```

Comment:
move this to `vertx-pg-client`

Reply:
We should probably use it to set the testContainers version in mysql-client as well instead of moving it to the pg-client. What do you think ?

Reply:
ok but use the version which MySQL client is using now otherwise we may need a CQ for upgrading.

Reply:
What is a CQ ?

Reply:
https://wiki.eclipse.org/Development_Resources/Contribution_Questionnaire